lib/output.js

```diff
@@ -694,6 +694,7 @@ function OutputStream(options) {
         output.semicolon();
     });
     DEFPRINT(AST_Toplevel, function(self, output){
+        use_asm = false;
```

Comment:
This isn't good code I guess, should be AST_Scope https://github.com/mishoo/UglifyJS2/blob/00b760363e529284e3077d924079ebaa915de7dd/lib/output.js#L422